src/utils/index.js

```diff
@@ -72,4 +72,4 @@ export const filterArticlesByDate = (articles) => {
     .sort((a, b) => new Date(b.createdAt) - new Date(a.createdAt))
     .slice(0, 6);
   return newArticles;
-};
+};
```

Comment:
Please put back the extra space for this

Reply:
Ok